crates/gpui/src/platform.rs

```diff
@@ -188,6 +188,7 @@ pub(crate) trait PlatformWindow: HasWindowHandle + HasDisplayHandle {
         answers: &[&str],
     ) -> Option<oneshot::Receiver<usize>>;
     fn activate(&self);
+    fn is_active(&self) -> bool;
```

Comment:
I don't think this is the correct approach, as implemented. We track active status at the cross-platform level, and adding a platform API that we then only use on initialization of a platform window feels incorrect. Let's instead change the return type of `Platform::open_window()` to return a `(Box<dyn PlatformWindow>, bool)`, with the second bool being the initial active status. That or we should change the existing `active` state checks in GPUI to use the true platform window checks.